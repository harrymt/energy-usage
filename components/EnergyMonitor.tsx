import React from 'react'

export const EnergyMonitor = ({}) => <div>Energy Monitor</div>

export default EnergyMonitor
