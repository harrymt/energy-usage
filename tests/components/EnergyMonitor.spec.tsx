import React from 'react'
import renderer from 'react-test-renderer'
import EnergyMonitor from '../../components/EnergyMonitor'

describe('EnergyMonitor', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<EnergyMonitor />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
