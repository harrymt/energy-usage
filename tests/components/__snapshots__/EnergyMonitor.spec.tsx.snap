// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EnergyMonitor should render correctly 1`] = `
<div>
  Energy Monitor
</div>
`;
